README.md

```diff
@@ -46,6 +46,7 @@ Here's how the default settings look like:
     static_url_base: 'https://ucarecdn.com',
     api_version: '0.5',
     cache_files: true,
+    store_files_upon_uploading: false,
```

Comment:
It is said in [Upload API docs](https://uploadcare.com/documentation/upload/#upload-body) that setting this flag is the same as not setting it at all (which earlier gem versions did), so adding this should be safe